src/defs_urls.h

```diff
@@ -53,6 +53,8 @@
     MIXXX_MANUAL_URL "/chapters/preferences.html#library"
 #define MIXXX_MANUAL_CUE_MODES_URL \
     MIXXX_MANUAL_URL "/chapters/user_interface.html#using-cue-modes"
+#define MIXXX_MANUAL_SYNC_MODES_URL \
+    MIXXX_MANUAL_URL "/chapters/user_interface.html#sync-and-rate-controls"
```

Comment:
Now that ___ has been merged, we should link to https://manual.mixxx.org/2.4/chapters/djing_with_mixxx#sync-lock-with-dynamic-tempo

Btw, when I click the (?) link I end up at https://manual.mixxx.org/2.4/chapters/user_interface.html#sync-and-rate-controls + X * PageDown (at the effects section)